### DISCUSSION TITLE
table什么时候可以支持拖拽排序啊

### DISCUSSION THREAD

**Discussion by CelesteCC** [vbenjs/vue-vben-admin#2523] [Q&A]
table什么时候可以支持拖拽排序啊

**Comment by WitMiao** [comment]
用[sortablejs](http://sortablejs.github.io/Sortable/)就好了，项目里封装了[useSortable](https://github.com/vbenjs/vue-vben-admin/blob/main/src/hooks/web/useSortable.ts)方法，页面引用就可以用。
参数传个dom节点和options参数，数据处理在onEnd事件内处理。

**Comment by lzdjack** [comment]
对暗黑主题没要的话  试试vxetable吧 vebn内部已经封装好了

**Reply by CelesteCC** [reply]
有事例吗，代码没有搜到vxetable

**Reply by lzdjack** [reply]
更新到最新代码，事例看官方dome，在 组件->table-vxetable
